cmd/snap/cmd_changes.go

```diff
@@ -122,14 +125,55 @@ func (c *cmdChanges) Execute(args []string) error {
 
 func (c *cmdTasks) Execute([]string) error {
 	cli := Client()
-	return showChange(cli, c.Positional.ID)
+	var id changeID
+	if c.Positional.ID == "" && c.LastChangeType == "" {
+		return fmt.Errorf(i18n.G("please provide change ID or type with --last=<type>"))
+	}
+	if c.Positional.ID != "" && c.LastChangeType != "" {
+		return fmt.Errorf(i18n.G("change use ID and type together"))
```

Comment:
the error message is somewhat confusing for me, how about reusing the exact same error message above? (perhaps emphasise the `either` by saying `please provide either change ID or change type with --last=<type>`